docs/spec/sections/mandatory-to-implement-algorithms.html

```diff
@@ -1,12 +1,24 @@
 <section class="normative">
   <h3>Mandatory Algorithms </h3>
   <p>
-    The following digital signature algorithms 
+    The following digital signature algorithms
     MUST be implemented by conforming issuers and verifiers.
   </p>
   <ul>
     <li>
-      <a data-cite="RFC7518#section-3.4">ES384</a>
+      <a data-cite="RFC7518#section-3.4">ES256</a>
     </li>
   </ul>
+<h3>Requirements</h3>
+<ul>
+<li>SHALL implement the [W3C-VC-DATA-MODEL] using the JSON-LD Compacted Document Form</li>
+<li>SHALL implement the enveloping proof mechanism defined in [W3C-VC-JOSE-COSE] with JOSE (Section 3.1.1)</li>
+<li>SHALL implement [W3C-BITSTRING-STATUS-LIST] for credential status checks including revocation checks</li>
+<li>SHALL implement [TRACE-API]</li>
+<li>SHALL implement [TRACE-VOCAB]</li>
+<li>SHALL implement [W3C-DID-CORE]</li>
```

Comment:
If I understand what you're trying to do, I think this will get closer to desired result. Check PR-Preview.
```suggestion
<li>SHALL implement the [[[W3C-VC-DATA-MODEL]]] [[W3C-VC-DATA-MODEL]] using the JSON-LD Compacted Document Form</li>
<li>SHALL implement the enveloping proof mechanism defined in [[[W3C-VC-JOSE-COSE]]] [[W3C-VC-JOSE-COSE]]with JOSE (Section 3.1.1)</li>
<li>SHALL implement [[[W3C-BITSTRING-STATUS-LIST]]] [[W3C-BITSTRING-STATUS-LIST]] for credential status checks including revocation checks</li>
<li>SHALL implement [[[TRACE-API]]] [[TRACE-API]]</li>
<li>SHALL implement [[[TRACE-VOCAB]]] [[TRACE-VOCAB]]</li>
<li>SHALL implement [[[W3C-DID-CORE]]] [[W3C-DID-CORE]]</li>
```

Reply:
^ yes, although I suspect we may need to update local bib to make the references bind, @nissimsan see:

https://github.com/w3c/respec/wiki/localBiblio

Make sure to pin the versions to the following:

The static versions may be dereferenced at the following links:
 - Verifiable Credentials Data Model v2.0 - https://www.w3.org/TR/2024/CR-vc-data-model-2.0-20240201/
 - Securing Verifiable Credentials using Jose and Cose - https://www.w3.org/TR/2024/WD-vc-jose-cose-20240126/
 - Bitstring Status List v1.0 - https://www.w3.org/TR/2024/WD-vc-bitstring-status-list-20240107/

Reply:
We don't want the references to float to latest version.

Reply:
Would approve after this change

Reply:
Thanks @TallTed, good stuff

Reply:
But none applied?